docs/archive-creation.md

```diff
@@ -0,0 +1,69 @@
+# Create plugin archive
+
+This action can be used to create plugin archives in a controlled and isolated environment via GitHub Actions.
+
+To achieve that, the reusable workflow:
+
+1. Installs dependencies (including dev-dependencies) defined in `composer.json`
+2. Executes `inpsyde/composer-assets-compiler` if required & configured by the package
+3. Executes `inpsyde/wp-translation-downloader` if required & configured by the package
+4. Re-installs composer dependencies without dev-dependencies
+5. Sets current commit hash and plugin version in the plugin's main file
+6. Runs `wp dist-archive` to create the final archive (with builtin support for a `.distignore` file)
+7. Uploads it as an artifact for download or further processing
+
+
+## Simple usage example:
+
+```yml
+name: Create release package
+on:
+  workflow_dispatch:
+    inputs:
+      PACKAGE_VERSION:
+        description: 'Package Version'
+        required: true
+jobs:
+  create_archive:
+    uses: inpsyde/reusable-workflows/.github/workflows/build-plugin-archive.yml@main
+    secrets:
+      COMPOSER_AUTH_JSON: ${{ secrets.PACKAGIST_AUTH_JSON }}
+    with:
+      PLUGIN_MAIN_FILE: ./my-plugin.php
+      PLUGIN_VERSION: ${{ inputs.PACKAGE_VERSION }}
+      #EXTRA_PHP_FILE: ./config/archive.php
+      EXTRA_PHP: |
+        echo 'hello world!';
+
+```
+
+## Configuration parameters
+
+### Inputs
+
+| Name                  | Default                                                       | Description                                                                    |
```

Comment:
The table Markdown syntax is malformed. Also, we don't include the full stops (`.`) in the description (applies to the other table as well).